db/migrate/20191111041659_create_jobs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateJobs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :jobs do |t|
+      t.text :content
+      t.integer :Tenant_id
```

Comment:
I believe we're using tenant email as a unique identifier, not tenant_id. I it'd be best if the second field was the tenant email instead of id but it would be good to check with the team on that to be 100% sure. Also might be good to include landlord email, vendor email but we can work on flushing out the jobs table as another pr.